catalog/app/containers/Bucket/Dir.js

```diff
@@ -145,6 +146,10 @@ export default function Dir({
           {renderCrumbs(getCrumbs({ bucket, path, urls }))}
         </div>
         <M.Box flexGrow={1} />
+        <FileView.ZipDownloadForm
+          suffix={`dir/${bucket}/${path}`}
+          label="Download dir as .zip"
```

Comment:
Instead of the down arrow, let's use the `file-zip` (I added it to our fort-awesome) or equivalent. It's just a zip archive icon. That way we can have less text in the button.

```suggestion
          label="Download directory"
```